20250226-actors-pubsub.md

```diff
@@ -0,0 +1,158 @@
+# Actor PubSub Implementation Proposal
+
+- Author(s): @joshvanl
+
+## Overview
+
+This proposal introduces PubSub capabilities for Dapr Actors, enabling actors to subscribe to messages for their actor type.
+Additionally, actors of a that type have the ability to publish messages to that type, to be delivered to  particular actor IDs.
+A single designated PubSub component will be marked as the actor PubSub for that namespace, similar to how an actor state store is defined.
+There can be only one actor PubSub Component per namespace.
```

Comment:
While having a means to set up a standing subscription like this is useful to subscribe all actors of a given type, I'd also like to see actors able to set up something similar to streaming subscriptions in which specific actor instances of a type can subscribe to a named pubsub component type, a variable topic name and assign their own filter (filtered at the runtime) so the actor is only activated on a match.

Reply:
Opening up the possibility for a streaming connection per actor ID that outlives the actor instance itself makes me a little bit uncomfortable. We should look into how this would scale.